packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -461,10 +466,15 @@ async function resolutions({ file, args = {} }) {
     pngCompressionLevel: 9,
     grayscale: false,
     duotone: false,
+    trace: {
```

Comment:
Would be awesome to implement `background` and `color` and expose these as configuration.

Reply:
👍 👍 Planning to expose all of those: https://github.com/tooolbox/node-potrace#parameters – `background` and `color` are already working:

![pastedgraphic-1](https://user-images.githubusercontent.com/21834/31574551-692bbf4e-b0d2-11e7-99d8-c32a6dafed90.png)